tests/mock/dist.rs

```diff
@@ -414,7 +414,7 @@ impl MockDistServer {
                 .iter()
                 .map(|v| toml::Value::String((**v).to_owned()))
                 .collect();
-            toml_profiles.insert(profile.to_string(), toml::Value::Array(array));
+            toml_profiles.insert((*profile).to_string(), toml::Value::Array(array));
```

Comment:
What about using String::from?

Reply:
Probably nothing, I'll admit I went with the recommendation from the compiler :D

Reply:
So str::to_string() use String::from under the hood: https://doc.rust-lang.org/nightly/src/alloc/string.rs.html#2179-2184.
I would file an issue in clippy in this case.

Reply:
I'll leave myself a TODO unless you want to do it.

Reply:
I don't bother me that much. Just a style.

Reply:
Fair enough, I'll bear it in mind next time clippy talks to me in that way :D